src/octoprint/server/util/sockjs.py

```diff
@@ -449,6 +449,7 @@ def on_printer_send_current_data(self, data):
                 "logs": self._filter_logs(logs),
                 "messages": messages,
                 "busyFiles": busy_files,
+                "markings": self._printer.get_markings(),
```

Comment:
This is sending all markings, every time. Once the mentioned cutoff is implemented, that shouldn't be such a big of an issue resource wise, however it still diverts from the per-connection backlog implementation used for temperatures, messages and logs and thus should probably be aligned.